third_party/terraform/tests/resource_cloudbuild_trigger_test.go

```diff
@@ -151,6 +150,19 @@ resource "google_cloudbuild_trigger" "build_trigger" {
       args = ["build", "-t", "gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA", "-f", "Dockerfile", "."]
       timeout = "300s"
     }
+    artifacts {
```

Comment:
In addition to this test, I'd recommend also adding these new fields to the cloudbuild_trigger_build.tf.erb example.